vos/vos/vos.py

```diff
@@ -2451,27 +2530,27 @@ def delete(self, uri):
             response = self.conn.session.delete(url)
             response.raise_for_status()
 
-    def get_info_list(self, uri):
```

Comment:
I see that you have removed the client.get_info_list method.  but this method is an exposed method so you are changing the public API of the client.  I think you should likely just add a deprecated property to this method but leave the original intact and add a completely distinct new method.